script/Deploy.sc

```diff
@@ -0,0 +1,96 @@
+import sys.process._
+import ammonite.ops._
+
+def ensureSuccess(status: Int): Unit =
+  if (status == 0) ()
+  else sys.error(s"Process exited with status $status")
+
+def run(args: String*): Unit = ensureSuccess(Process(args.toList).!)
+def runD(args: String*)(dir: Path): Unit = {
+  println(s"Running command ${args.mkString(" ")} in working directory $dir")
+  ensureSuccess(Process(args.toList, Some(dir.toIO)).!)
+}
+def runSlurp(args: String*): String =
+  Process(args.toList).lineStream.toList.headOption.getOrElse("")
+
+/**
+  * @param dataTarball   Path of the packaged (.zip) `data` application
+  * @param serverTarball Path of the packaged (.zip) `server` application
+  * @param reposDir      Directory that contains the `scaladex-credentials`, `scaladex-index` and
+  *                      `scaladex-contrib` sub-directories containing the git repositories
+  */
+@main def main(dataTarball: String, serverTarball: String, reposDir: String) = {
```

Comment:
This is the deployment script to run on the server. We would eventually move it to an sbt task, but I leave it to another PR.